modules/tensor_mechanics/test/src/materials/SS316HLAROMANCEStressUpdateTest.C

```diff
@@ -7,69 +7,77 @@
 //* Licensed under LGPL 2.1, please see LICENSE for details
 //* https://www.gnu.org/licenses/lgpl-2.1.html
 
-//  saturation as a function of effective saturation, and its derivs wrt effective saturation
-//
-#include "SS316LAROMData.h"
+#include "SS316HLAROMANCEStressUpdateTest.h"
 
-registerMooseObject("TensorMechanicsApp", SS316LAROMData);
+registerADMooseObject("TensorMechanicsTestApp", SS316HLAROMANCEStressUpdateTest);
 
-template <>
-InputParameters
-validParams<SS316LAROMData>()
-{
-  InputParameters params = validParams<LAROMData>();
-  params.addClassDescription("UserObject that stores the relevant data for the reduced order model "
-                             "for SS316. Must be used in conjuction with ADROMCreepStressUpdate");
-  return params;
-}
-
-SS316LAROMData::SS316LAROMData(const InputParameters & parameters) : LAROMData(parameters) {}
-
-unsigned int
-SS316LAROMData::getStressIndex() const
-{
-  return 2;
-}
-
-unsigned int
-SS316LAROMData::getDegree() const
-{
-  return 4;
-}
+defineADValidParams(
+    SS316HLAROMANCEStressUpdateTest,
+    ADLAROMANCEStressUpdateBase,
+    params.addClassDescription("LAROMANCE creep update model for SS316H");
 
-Real
-SS316LAROMData::getMaxRelativeMobileInc() const
-{
-  return 0.5;
-}
-
-Real
-SS316LAROMData::getMaxRelativeImmobileInc() const
-{
-  return 0.5;
-}
+    // Override defaults for material specific parameters below
+    params.addRangeCheckedParam<Real>("initial_mobile_dislocation_density",
+                                      5.0e12,
+                                      "initial_mobile_dislocation_density >=1.32776E+12 & "
+                                      "initial_mobile_dislocation_density <= 9.99959E+12",
```

Comment:
Are these rather specific, and new, range numbers based on additional training of the ROM that Laurent, Arul, and Aaron have done with this model?

Reply:
These are specific to the material and how it is processed. The values used here are suggested, and good default values.

Reply:
Topher, please allow me to apologize if my comment came across as confrontational or overly pickly, I was just curious about the specific numbers. Thank you for the explanation

Reply:
No worries! It was a fair question, and I know we are on the same team :)

Reply:
There's a lot of insignificant digits there...